jena-arq/src/main/java/org/apache/jena/sparql/graph/GraphOps.java

```diff
@@ -29,60 +29,49 @@
 import org.apache.jena.sparql.core.Quad ;
 
 // Combine with Jena GraphUtils.
-public class GraphOps
-{
-    
-    public static boolean containsGraph(DatasetGraph dsg, Node gn)
-    {
-        // [[DynDS]]
-        if ( Quad.isDefaultGraph(gn))
+public class GraphOps {
+    public static boolean containsGraph(DatasetGraph dsg, Node gn) {
+        if ( Quad.isDefaultGraph(gn) )
             return true ;
-        if ( Quad.isUnionGraph(gn))
+        if ( Quad.isUnionGraph(gn) )
             return true ;
         return dsg.containsGraph(gn) ;
     }
-    
-    public static Graph getGraph(DatasetGraph dsg, Node gn)
-    {
-        // [[DynDS]]
+
+    public static Graph getGraph(DatasetGraph dsg, Node gn) {
         if ( gn == null )
             return dsg.getDefaultGraph() ;
         if ( Quad.isDefaultGraph(gn) )
             // Explicit or generated.
             return dsg.getDefaultGraph() ;
-        if ( Quad.isUnionGraph(gn))
+        if ( Quad.isUnionGraph(gn) )
             return unionGraph(dsg) ;
         return dsg.getGraph(gn) ;
     }
-    
-    public static Graph unionGraph(DatasetGraph dsg)
-    {
+
+    public static Graph unionGraph(DatasetGraph dsg) {
+        // Snapshot it now.
```

Comment:
How come a snapshot here? I have no objection to it (other than the cost of rolling up), just wondering. Is it because of potential `ConcurrentModificationException`s?